opc-ua-sdk/sdk-client/src/main/java/org/eclipse/milo/opcua/sdk/client/api/subscriptions/UaSubscription.java

```diff
@@ -272,6 +301,12 @@
 
     }
 
+    interface ItemTransferredCallback {
+
+        void onItemTransferred(SerializationContext serializationContext, UaMonitoredItem item, int clientHandle);
```

Comment:
Is the `clientHandle` parameter necessary? Can't you get it just by calling `item.getClientHandle()`?

Reply:
Yeah, you're right. 

I modeled this off of the `createMonitoredItems` method, but didn't have a use case to provide an index like that method does. Do you think we need an index there? If not I'm good with just removing the `clientHandle`.

Reply:
Modified callback so it provides an index instead of client handle, but still not sure it's actually useful for anything.